### PR TITLE
Scarves in Loadouts

### PR DESCRIPTION
**Pull request by kryopteryx**
## About the PR
Added scarves to loadouts for the following jobs/departments, minus Command:
- Cargo/Salvage
- Engineering/Atmos
- Medical
- Mime
- Science
- Security

## Why / Balance
Most of these are already available in the respective department's clothing vend, this just makes you able to spawn in with them using loadouts. Variations for cargo/salvage, robotics and paramedic are intended to match the jump-/hardsuits' colors.
Excluded Command because their capes/cloaks add to their recognizability.

## Technical details
Created loadout groups where necessary, and/or added the specific items to those jobs/groups.

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: Added scarves to loadouts for various jobs.

### CODE REVIEW

**Review by Tayrtahn — CHANGES_REQUESTED** [review]
Yeah, you can't have multiple `loadout` or `startingGear` prototypes with the same name. Being in separate yml files doesn't matter; as far as the parser is concerned, they're all in one big file.

Follow the example of how glasses are set up in loadouts: make a scarves.yml file in the Miscellaneous folder of loadouts and define all the scarf `loadout`s and `startingGear`s there (removing them from the department-specific files).

**Comment by Tayrtahn** [comment]
On a side note, I appreciate that you didn't just throw all of the scarves in as options for everyone. Matching the department colors is a nice touch!

**Comment by kryopteryx** [comment]
I mainly just copy & pasted how ties have been configured so that explains it lol. Since some jobs already have neck slot options I've added the scarves to the backpack (like trinkets are handled) to avoid potential bugs. It's not an elegant solution but I hope it works. Do let me know if I missed anything!

**Comment by kryopteryx** [comment]
Requesting a review to check if the general changes are correct. I'm a bit lost on what's up with the blue scarf, since it uses the same ID as in `\Entities\Clothing\Neck\scarfs.yml` just like all the others and is set up the same way, unless I'm missing something...

File: Resources/Prototypes/Loadouts/Jobs/Security/detective.yml [hunk 1]
```diff
@@ -16,16 +16,6 @@
...
-  
-# Neck
-- type: loadout
-  id: DetectiveTie
-  equipment: DetectiveTie
-
-- type: startingGear
-  id: DetectiveTie
-  equipment:
-    neck: ClothingNeckTieDet
```
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/Loadouts/Jobs/Security/detective.yml:28]: why does this just remove the detective tie for no reason

Comment (kryopteryx) [reply Resources/Prototypes/Loadouts/Jobs/Security/detective.yml:28]: that is def not intended oof, will fix

**Comment by UbaserB** [comment]
Can you let medical doctors optionally have dark blue scarf along with the light version, and mimes have black?

**Comment by Plykiya** [comment]
Librarians need green scarves

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
You shouldn't be adding coloured scarves as it's just unnecessary bloat, just wait for loadout recolouring.

**Comment by ColdDead** [comment]
Can you add the senior stuff for paramedics to? some peopel like me pref paramedic gameplay, But have the knowledge of a senior

**Comment by Emisse** [comment]
Closed due to feature freeze May 10th-June 14th. Comment to have it reopen after this.

**Pull request closed without merging**